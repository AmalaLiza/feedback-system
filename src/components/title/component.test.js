import React from "react";
import { shallow } from "enzyme";
import Title from "./component";
import renderer from "react-test-renderer";

const props = {
	title: "Some random title",
};

function setup() {
	const wrapper = shallow(<Title {...props} />);
	return { wrapper, props };
}

describe("Title Component Test Suite", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<Title {...props} />).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it("Should have a h1 element", () => {
		const { wrapper } = setup();
		expect(wrapper.find("h1").exists()).toBe(true);
	});
});
