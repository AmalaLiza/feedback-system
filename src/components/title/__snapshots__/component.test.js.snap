// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component Test Suite renders correctly 1`] = `
<h1
  className="title"
>
  Some random title
</h1>
`;
